addons/microsoft_outlook/models/microsoft_outlook_mixin.py

```diff
@@ -78,8 +78,12 @@ def open_microsoft_outlook_uri(self):
         if not self.env.user.has_group('base.group_system'):
             raise AccessError(_('Only the administrator can link an Outlook mail server.'))
 
-        if not self.use_microsoft_outlook_service or not self.is_microsoft_outlook_configured:
-            raise UserError(_('Please configure your Outlook credentials.'))
+        if not self.use_microsoft_outlook_service or not self.is_microsoft_outlook_configured or not self.smtp_user:
+            raise UserError(_(
+                        'Please configure your Outlook credentials. '
+                        'This server can only be used for your outlook/office365 email address. '
+                        'Please fill the "Username" field with your outlook/office365 username (your email address). '
+                        'This should be the same account as the one used for the Outlook OAuthentication Token.'))
```

Comment:
Don't forget to add an entry to the pot for this (export a new one and copy/paste)

Reply:
Did I just right now and hopefully I did it right (will push commits once I tackle the rest of your feedback).